NAMESPACE

```diff
@@ -12,6 +12,13 @@ export(florabank_taxon_ifbl_year)
 export(florabank_traits)
 export(gbif_species_name_match)
 export(guess_projection)
+export(inboveg_classification)
+export(inboveg_coverage)
```

Comment:
The current build is failing because `inboveg_coverage()`, `inboveg_metadata_kop()`, `inboveg_synonym_list()` are exported but not defined.

Reply:
Thanks for reporting. Should be solved in 3e01890.